whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -2554,6 +2568,98 @@ class PostgreSQLComponent {
             }
         }
     }
+    
+    enum NotificationType {
+        dependency_cache_invalidate // LISTEN silently fails to register if upper case(???)
```

Comment:
Given https://stackoverflow.com/a/5173993 I'm guessing the channel name here is _not_ forcibly lowercased
```
try (PreparedStatement statement = getMyConnection().prepareStatement('SELECT pg_notify(?, ?)')) {
    statement.setString(1, type.toString())
```
while here it _is_
```
statement.execute("LISTEN ${t.toString()}")
```
?

Reply:
Ah!

I'll make that clear.

Reply:
Forgot to push it here....
On `develop` now https://github.com/libris/librisxl/commit/8a047738c4ae3ac9cbfd75d209009a6cd697fef8